ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Ledger/NetworkProtocolVersion.hs

```diff
@@ -67,6 +62,7 @@ instance SupportedNetworkProtocolVersion (ShelleyBlock proto era) where
       , (NodeToClientV_15, ShelleyNodeToClientVersion7)
       , (NodeToClientV_16, ShelleyNodeToClientVersion8)
       , (NodeToClientV_17, ShelleyNodeToClientVersion9)
+      -- , (NodeToClientV_18, ShelleyNodeToClientVersion10)
```

Comment:
Why is this commented out? Just a WIP thing?

Reply:
Shelley...Version10 constructor is added in the next PR with Damian's commits

Reply:
Cherry-picking from #1251/merging #1251 into this PR sounds useful then :+1: